src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
 export interface Figure {
-
+  color: string;
+  shape: string;
```

Comment:
```suggestion
  color: string;
  shape: string;
```
Would be great to create enums for colors and shapes